src/ducks/transactions/TransactionCategoryEditor.jsx

```diff
@@ -9,32 +9,56 @@ import CategoryChoice from 'ducks/categories/CategoryChoice'
 /**
  * Edits a transaction's category through CategoryChoice
  */
-const TransactionCategoryEditor = props => {
+const TransactionCategoryEditor = ({
+  transaction,
```

Comment:
We can used `transactions` here, it avoids to check if is an array (so removed `Array.isArray(transaction) ? transaction : [transaction]`). 

And to pass [transaction] in `transactions`component props.

Reply:
Oui, y'avait une très bonne raison du pourquoi je ne l'ai finalement pas fait... mais là je ne vois plus ce que c'était :p